portfolio/src/main/webapp/script.js

```diff
@@ -127,3 +127,23 @@ const slideToggle = (elementId, duration)  => {
     slideUp(elementId, duration);
   }
 }
+
+/* fetches comments content from webserver and adds to DOM in container with id elementID */
+const addToDOM = (elementId) => {
+  fetch("/comments").then(response => response.json()).then((comments) => {
+
+    // add each parsed json.commentText to the DOM
+    const destinationDiv = document.getElementById(elementId);
+    
+    for (comment in comments) {
```

Comment:
```
comments.map(comment => comment.text).forEach(createSpan);
```

Reply:
done!

Reply:
It's a little longer in mine because needed to appendChild too